docs-src/docs/setup/upgrade.md

```diff
@@ -0,0 +1,43 @@
+# Upgrade Layer0
+
+This section provides procedures for upgrading your Layer0 installation to the latest version.
+
+1. In the [Downloads section of the home page](/index.html#download), select the appropriate installation file for your operating system. Extract the zip file to a directory on your computer, and then move the **l0** and **l0-setup** files to a folder in your system path, replacing any previous versions of these files.
+
+2. Follow the instructions for creating an [Administrator Access Key](install/#part-2-create-an-access-key). You will use this access key when l0-setup prompts you for AWS credentials.
```

Comment:
Hm. We don't really need this step 2 at all in its current form, do we? Shouldn't this just be "run `l0-setup endpoint` to set up your environment so you can run the `apply`"?